keyboards/gck/num3x3/readme.md

```diff
@@ -0,0 +1,25 @@
+# num 3x3
+
+num 3x3
+
+* Keyboard Maintainer: [Giliy666](https://github.com/Giliy666)
+* Hardware Supported: rp2040, stm32f401
+* Hardware Availability: [aliexpress](https://aliexpress.ru/)
```

Comment:
Linking to ali rather than a hardware repo, or site which sells the product seems fishy? Like its a handwired keyboard and should be relocated to within the `keyboards/handwired/` folder?

Can you provide images per the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1#L3) so we can verify.